lametro/bills.py

```diff
@@ -50,8 +50,12 @@ def sponsorships(self, matter_id) :
     def actions(self, matter_id) :
         old_action = None
         for action in self.history(matter_id) :
-            action_date = action['MatterHistoryActionDate']
+            # Metro admin added an action with a classifcation of 'DISCUSSED (do not use).'
+            # They requested that we do not pull in this data.
             action_description = action['MatterHistoryActionName'].strip()
+            if 'do not use' in action_description:
+                action_description = None
```

Comment:
Great comment! I understand this pattern, but I think it would be more explicit to use `continue` here rather than artificially setting the description to `None`. What do you think?

Reply:
I prefer "continue"! Thanks.